code/datums/scenarios/scenario.dm

```diff
@@ -0,0 +1,132 @@
+/singleton/scenario
+	/// The scenario's name. Displayed in the round end report and some other places.
+	var/name = "Generic Scenario"
+	/// The scenario's description. Displayed in the round end report and some other places.
+	var/desc = "A generic scenario that should not be in the rotation."
+	/// What sectors this scenario can spawn in. An empty list is all sectors.
+	var/list/sector_whitelist = list()
+	/// The type of scenario this is. NOT a boolean or a bitfield.
+	var/scenario_type = SCENARIO_TYPE_NONCANON
+
+	// All scenarios happen in an away site, which is manually spawned the very second the round starts.
+	// This away site can have its exoplanet themes defined and then mapped to give the semblance of being on a planet (see the Tret away site).
+	// Alternatively, without defining exoplanet themes, it can take place in space, for example.
+	// In the future there will likely be support for no-away site scenarios that take place on the Horizon.
+
+	/// The away site ID of the scenario to spawn. This is what you should edit, set it to the away site's id variable.
+	var/scenario_site_id
+	/// This is the away site where the scenario takes place. Do not edit this variable - it's set automatically.
+	var/datum/map_template/ruin/away_site/scenario_site
+	/// Weight given to the odyssey in pickweight when being picked.
+	var/weight = 20
+	/// The minimum amount of total players required for an odyssey to spawn.
+	var/min_player_amount = 8
+	/// The minimum amount of actors we want to spawn in this odyssey.
+	var/min_actor_amount = 1
+
+	/// The title for the messages sent to the Horizon to notify them of the scenarios, both in notify_scenario_early and notify_scenario_late.
+	var/horizon_announcement_title = "Central Command Situation Report"
+	/// The announcement message sent to the Horizon immediately after roundstart (5 minutes or so), telling them to prepare for a yet unknown expedition.
+	var/horizon_early_announcement_message = "SCCV Horizon, your sensors suite has located a site of interest and we are in the process of scanning the received deta. You will shortly be sent to investigate it. Please prepare an expedition."
+	/// The announcement message sent to the Horizon around 20 minutes in, typically telling them to go investigate the scenario and the reason why.
+	var/horizon_late_announcement_message = "The site of interest has been located and its coordinates have been registered on your sensors. Send an expedition to investigate."
+
+	/// The announcement message for offships. This one contains all the info and is sent quickly.
+	var/offship_announcement_message = "A recent sensors scan indicates the presence of a site of interest to investigate. The oordinates have been registered on the sensors consoles."
```

Comment:
Should probably be its own singleton/datum announcement thing instead of a bunch of vars thrown in, and then referenced by the scenario?

Reply:
huh?

Reply:
Yes, but it would be logically grouped, instead of a bunch of vars thrown in